app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -67,24 +71,34 @@
     @Persisting(1)
     @MetaField(META_XMLNS)
     private String xmlns;
+
     @Persisting(2)
     @MetaField(META_INSTANCE_URI)
     private String instanceURI;
+
     @Persisting(3)
     @MetaField(META_STATUS)
     private String status;
+
     @Persisting(4)
     private byte[] aesKey;
+
     @Persisting(value = 5, nullable = true)
     @MetaField(META_UUID)
     private String uuid;
+
     @Persisting(6)
     @MetaField(META_LAST_MODIFIED)
     private Date lastModified;
+
     @Persisting(7)
     @MetaField(META_APP_ID)
     private String appId;
 
+    @Persisting(value = 8, nullable = true)
+    @MetaField(META_SUBMISSION_ORDERING_NUMBER)
+    private String submissionOrderingNumber;

```

Comment:
why is this being stored as a string which is turned into a number, btw? is that a limitation of the metadata framework?